code/controllers/subsystem/icon_smooth.dm

```diff
@@ -30,15 +29,14 @@ SUBSYSTEM_DEF(icon_smooth)
 			can_fire = 0
 
 /datum/controller/subsystem/icon_smooth/Initialize()
-	smooth_zlevel(1,TRUE)
-	smooth_zlevel(2,TRUE)
-	var/queue = smooth_queue
+	var/list/queue = smooth_queue
 	smooth_queue = list()
 	for(var/V in queue)
```

Comment:
Shouldn't this be while(length(queue))?

Reply:
I mean it could be, that would avoid a list copy which is nice, but it really isn't that important because the list shouldn't have anything added to it while it runs